src/Rules/Install.hs

```diff
@@ -0,0 +1,299 @@
+{-# LANGUAGE OverloadedStrings, FlexibleContexts #-}
+module Rules.Install (installRules) where
+
+import Base
+import Target
+import Context
+import qualified Predicate
+import Predicate hiding (builder)
+import Settings
+import Settings.Path
+import Util
+import GHC
+import Rules
+import Rules.Wrappers (WrappedBinary(..), Wrapper, installWrappers)
+import Rules.Libffi
+import Rules.Generate
+import Settings.Packages.Rts
+import Oracles.Config.Setting
+import Oracles.PackageData
+import Oracles.Path
+
```

Comment:
We need an overview comment somewhere here describing what's going on. At minimum, please describe the resulting tree structure, linking it to various path/dir settings.